### DISCUSSION TITLE
Welcome to FShopOnWeb Discussions!

### DISCUSSION THREAD

**Discussion by seangwright** [NitroDevs/FShopOnWeb#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KyleMcMaster** [comment]
Great idea to use discussions as the official communication channel for this project. Hopefully this will be helpful in getting feedback from the F# community in the future as well as serving as a historical record of the decisions made along the way.
